.github/workflows/firefox-ios-update_remote_settings_data_script.yml

```diff
@@ -0,0 +1,75 @@
+name: Fetch Remote Settings
+
+# Remote settings are updated every Tuesday and Thursday
+
+on:
+  schedule:
+    - cron: '0 0 * * 2' # Every Tuesday at midnight
+    - cron: '0 0 * * 4' # Every Thursday at midnight
+  workflow_dispatch: # Allows manual triggering of the workflow
+
+env:
+    SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+jobs:
+  fetch-remote-settings:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.7]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get the current GitHub Action name
+        run: |
+          github_action_name="Fetch Remote Settings"
+          echo "github_action_name=$github_action_name" >> $GITHUB_ENV
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install requests
+
+      - name: Fetch remote settings and update files
+        run: python ./firefox-ios/Client/Assets/RemoteSettingsData/Update_Remote_Settings.py
+
+      - name: Commit and push changes
+        run: |
+          git diff
+          git diff --quiet || (git add firefox-ios/Client/Assets/RemoteSettingsData/RemotePasswordRules.json && git commit -m "Update remote settings with latest rules" && git push)
```

Comment:
@isabelrios also added specific json file location, tagging you to double check

Reply:
You would need to remove the latest part: `&& git commit -m "Update remote settings with latest rules" && git push` 
The github action [fails](https://github.com/isabelrios/firefox-ios/actions/runs/10699097190/job/29659886723) to do the push directly.

You have the next step to create the [PR](https://github.com/isabelrios/firefox-ios/pull/250) with the changes to review and then merge.

Reply:
@isabelrios let me DM you as I am wondering why do I even need the commit part considering we are doing the create pull request